src/components/Form.js

```diff
@@ -0,0 +1,10 @@
+const Form = () => (
+  <>
+    <form>
+      <input type="text" placeholder="Book title" />
+      <input type="text" placeholder="Author" />
+    </form>
```

Comment:
- [ OPTIONAL ] Great job so far 👏🏻👏🏻 but I recommend to add a button in your form as well so when a user want to submit a new book can press on it 💪🏻 I assign it as optional because it is not explicit in the requirements but it is very important to add so in the future want to add a functionality for submmiting to the button 💪🏻🔥🤩✅✨